game/resource/English/ability/items/tooltip_drums_of_endurance_oaa.txt

```diff
@@ -0,0 +1,49 @@
+//===================================================================
+// Drums of Endurance
+//===================================================================
+
+"DOTA_Tooltip_Ability_item_drums_of_endurance_2"                          "Drum of Endurance"
```

Comment:
Should use references to existing vanilla keys like #{DOTA_Tooltip_ability_item_ancient_janggo}. The note on charges should also be added to level 1 drums. Not sure if it's better to put that in this file or the ancient_janggo file.

Reply:
-Not Sure whether to change it or not. Any difference/advantage for using vanilla keys?
-Will Add the charges note to level 1 drums.
-It can be in either, just thought it would be better to segregate the base default drums and the custom oaa drums

Reply:
Levelled items should just refer to keys for the base level unless for some reason a specific level needs different text. Cause if text needs to be changed in future it can be done in one place. And in the case of vanilla text, the text will automatically update to follow any changes in vanilla.